### PR TITLE
Siddhi4 execution files

### PR DESCRIPTION
**Pull request by Ambujaa10**
### Proposed changes in this pull request

[List all changes you want to add here. If you fixed an issue, please
add a reference to that issue as well.]

- Siddhi4 execution files have been created for session data, long session alerts, suspicious login detection

### When should this PR be merged

[Please describe any preconditions that need to be addressed before we
can merge this pull request.]

### Follow up actions

[List any possible follow-up actions here; for instance, testing data
migrations, software that we need to install on staging and production
environments.]

-

#### General

- [ ] **Is this PR explained thoroughly?** All code changes must be accounted for in the PR description.
- [ ] **Is the PR labeled correctly?**

#### Functionality

- [ ] **Are all requirements met?** Compare implemented functionality with the requirements specification.
- [ ] **Does the UI work as expected?** There should be no Javascript errors in the console; all resources should load. There should be no unexpected errors. Deliberately try to break the feature to find out if there are corner cases that are not handled.

#### Code

- [ ] **Do you fully understand the introduced changes to the code?** If not ask for clarification, it might uncover ways to solve a problem in a more elegant and efficient way.
- [ ] **Does the PR introduce any inefficient database requests?** Use the debug server to check for duplicate requests.
- [ ] **Are all necessary strings marked for translation?** All strings that are exposed to users via the UI must be [marked for translation](https://docs.djangoproject.com/en/1.10/topics/i18n/translation/).

#### Tests

- [ ] **Are there sufficient test cases?** Ensure that all components are tested individually; models, forms, and serializers should be tested in isolation even if a test for a view covers these components.
- [ ] **If this is a bug fix, are tests for the issue in place?**  There must be a test case for the bug to ensure the issue won’t regress. Make sure that the tests break without the new code to fix the issue.
- [ ] **If this is a new feature or a significant change to an existing feature?** has the manual testing spreadsheet been updated with instructions for manual testing?

#### Security

- [ ] **Confirm this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.**
- [ ] **Are all UI and API inputs run through forms or serializers?**
- [ ] **Are all external inputs validated and sanitized appropriately?**
- [ ] **Does all branching logic have a default case?**
- [ ] **Does this solution handle outliers and edge cases gracefully?**
- [ ] **Are all external communications secured and restricted to SSL?**

#### Documentation

- [ ] **Are changes to the UI documented in the platform docs?** If this PR introduces new platform site functionality or changes existing ones, the changes should be documented.
- [ ] **Are changes to the API documented in the API docs?** If this PR introduces new API functionality or changes existing ones, the changes must be documented.
- [ ] **Are reusable components documented?** If this PR introduces components that are relevant to other developers (for instance a mixin for a view or a generic form) they should be documented in the Wiki.

### CODE REVIEW

File: features/org.wso2.analytics.is.feature/src/main/siddhi4/Readme.rtf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1561\cocoasubrtf200
```
Comment (ruwanta) [inline features/org.wso2.analytics.is.feature/src/main/siddhi4/Readme.rtf]: Please make this as README.md, in markdown format.

File: features/org.wso2.analytics.is.feature/src/main/siddhi4/IsAnalytics-ExecutionPlan-SessionData.siddhi [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+on s.id == t.id AND s.meta_tenantId == t.meta_tenantId
+select s.meta_tenantId, s.bucketId, s.bucketId as bucketStart, (s.bucketId + 60000) as bucketEnd, time:extract(bucketId, 'year') as year, time:extract(bucketId, 'month') as month, time:extract(bucketId, 'day') as day, time:extract(bucketId, 'hour') as hour, time:extract(bucketId, 'minute') as minute,  (s.net  + ifThenElse(t.activeCount is null, convert(0, 'long'), t.activeCount) ) as activeSessionCount, s.newSessionCount, s.terminatedSessionCount
+group by s.meta_tenantId
+insert into SessionStatPerMinute;
+
+from SessionStatPerMinute#window.length(1)
+select meta_tenantId, 'count' as id, activeSessionCount as activeCount
+update or insert into activeSessionCountTable
+on activeSessionCountTable.meta_tenantId==meta_tenantId AND activeSessionCountTable.id==id;
+
```
Comment (ruwanta) [inline features/org.wso2.analytics.is.feature/src/main/siddhi4/IsAnalytics-ExecutionPlan-SessionData.siddhi]: we may not need empty spaces

**Review by ruwanta — CHANGES_REQUESTED** [review]
Please do changes marked.

File: features/org.wso2.analytics.is.feature/src/main/siddhi4/Readme.rtf (hunk 1 above)
Comment (Ambujaa10) [reply features/org.wso2.analytics.is.feature/src/main/siddhi4/Readme.rtf]: ok thank you

**Comment by Ambujaa10** [comment]
I made the changes accordingly.

**Review by ruwanta — APPROVED** [review]

**Comment by Ambujaa10** [comment]
changes made on README

**Pull request merged**